package.json

```diff
@@ -24,10 +24,10 @@
     "async": "0.2.9"
   },
   "devDependencies": {
-    "expresso": "^0.9.2",
-    "grunt": "^0.4.5",
-    "grunt-fh-build": "^0.5.0",
-    "istanbul": "^0.3.15",
+    "expresso": "~0.9.2",
+    "grunt": "~0.4.5",
```

Comment:
Because it was using `^` so it should have been installing the latest version of grunt which is `1.0.1`. Changing the version to `~0.4.5` should downgrade the version